.gitignore

```diff
@@ -34,4 +34,5 @@ package-lock.json
 *.png
 
 # flask eviroment variables
-*.flaskenv
\ No newline at end of file
+*.flaskenv
+results.sqlite
```

Comment:
this can be cleaned up since Redis is  serving as  a result backend

Reply:
done